feature: landing page and alert component

sunilvr opened the pull request.

sunilvr committed:
feature: landing page and alert component

sunilvr committed:
feature: code refactoring

sunilvr committed:
alerts and buttons - code refactoring

sunilvr committed:
feature : added code snippets

sunilvr committed:
feature: changed font-family

changed font family and edited installation guide

sunilvr committed:
code refactoring and added some new subcomponents

added link and floating buttons
responsiveness changes
alerts with read more links

sunilvr committed:
updated font-size of html for landing page

anik31 reviewed (approved):
Looks good. Add some space between alert icon & message.

abhisheksahani-nova commented on components/buttons/buttons.css:
```diff
@@ -0,0 +1,66 @@
+}
+
+.secondary-btn-outlined{
+    background-color: white;
```
Use variable everywhere for colors white and gray as well

sunilvr replied:
Hi Abhishek
thanks for the review. Changes suggested by you and Aniket have been incorporated now

sunilvr replied:
@abhisheksahani-nova Could you please approve the PR as well?

abhisheksahani-nova replied:
Yess go ahead

sunilvr committed:
PR review - fixed code as per comments

PR review - fixed code as per comments
added space between alert icons and messages
used css variables where ever missed for colors
